src/containers/InfoProyecto/index.js

```diff
@@ -39,134 +81,203 @@ export default function InfoProyecto({ projectData, type, state }) {
+
+            <Box mt={2}>
+              <Typography
+                style={{ fontWeight: 600 }}
+                variant="body1"
+                display="inline"
+                gutterBottom
+              >
+                Tipo proyecto:{" "}
               </Typography>
-            ) : (
               <Typography display="inline" variant="body1">
-                Indefinida
+                {type}
               </Typography>
-            )}
-          </Box>
+            </Box>
 
-          <Box mt={2}>
-            <Typography
-              style={{ fontWeight: 600 }}
-              variant="body1"
-              gutterBottom
-            >
-              Descripción:{" "}
-            </Typography>
-            <Typography display="inline" variant="body1">
-              {projectData.description}
-            </Typography>
-          </Box>
+            <Box mt={2} style={{ display: "flex", gap: "0.5rem" }}>
+              <Typography
+                style={{ fontWeight: 600, alignSelf: "center" }}
+                variant="body1"
+                display="inline"
+                gutterBottom
+              >
+                Estado:{" "}
+              </Typography>
+              <FormControl>
+                <Select
+                  labelId="state-input"
+                  label="Estado"
+                  id="project_state"
+                  name="project_state"
+                  style={{ width: "unset" }}
+                  value={project_state}
+                  onChange={(e) => {
+                    setProjectState(e.target.value);
+                  }}
+                >
+                  <MenuItem value="verde">{renderColor("verde")}</MenuItem>
+                  <MenuItem value="amarillo">
+                    {renderColor("amarillo")}
+                  </MenuItem>
+                  <MenuItem value="rojo">{renderColor("rojo")}</MenuItem>
+                  <MenuItem value="upcoming">
+                    {renderColor("upcoming")}
+                  </MenuItem>
+                </Select>
+              </FormControl>
+            </Box>
 
-          {projectData.budget ? (
             <Box mt={2}>
               <Typography
                 style={{ fontWeight: 600 }}
                 variant="body1"
                 display="inline"
                 gutterBottom
               >
-                Budget:{" "}
+                Fecha Inicio:{" "}
               </Typography>
               <Typography display="inline" variant="body1">
-                {projectData.budget}
+                {moment(projectData.start_date).format("DD/MM/YYYY")}
+              </Typography>
+            </Box>
+            <Box mt={2}>
+              <Typography
+                style={{ fontWeight: 600 }}
+                variant="body1"
+                display="inline"
+                gutterBottom
+              >
+                Fecha Fin:{" "}
+              </Typography>
+              {projectData.end_date ? (
```

Comment:
You can do the conditional stuff inside typograhy tag.
`<Typography display="inline" variant="body1">
                  {projectData.end_date ? moment(projectData.end_date).format("DD/MM/YYYY") : 'Indefinida'}
                </Typography>`